firebase-vscode/src/test/tsconfig.test.json

```diff
@@ -6,6 +6,7 @@
 		"outDir": "../../dist/test",
 		"sourceMap": true,
 		"rootDir": "./",
-		"strict": true,
-	}
+		"strict": true
+	},
+	"exclude": ["**/*.test.ts"]
```

Comment:
can you explain this? i'm confused why we're excluding here...

Reply:
`tsc` is only used to compile the 2 test scaffolding files (runTest.ts and index.ts). We don't want it to run on the src files, as it won't work without webpack. All the *.test.ts files should be run through webpack, not raw tsc. The tsc process (last step in `pretest`) uses the tsconfig to determine what files it compiles, the webpack process sort of overrides this by specifying specific `entry` files, and when you do that, `exclude` won't exclude anything those files import.